.circleci/config.yml.HIGHRES

```diff
@@ -2183,6 +2183,98 @@ jobs:
     - CCM_HEAP_NEWSIZE: 256M
     - JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64
     - JDK_HOME: /usr/lib/jvm/java-8-openjdk-amd64
+  utests_system_keyspace_directory:
+    docker:
+    - image: nastra/cassandra-testing-ubuntu1910-java11-w-dependencies:20200603
```

Comment:
this should use the same docker image as the other test targets (`apache/cassandra-testing-ubuntu2004-java11-w-dependencies:20210105`)

Reply:
Good catch!